index.html

```diff
@@ -2874,7 +2874,16 @@ <h3>The <dfn>EventSubscriptionHandler</dfn> callback</h3>
           <li>
             Set up the <a>WoT Interactions</a> based on introspecting ||td||
             as explained in [[!WOT-TD]] and [[!WOT-PROTOCOL-BINDINGS]].
-            Make a request to the underlying platform to initialize the <a>Protocol Bindings</a> and then start serving external requests for <a>WoT Interactions</a> (read, write and observe <a>Properties</a>, invoke <a>Action</a>s and manage <a>Event</a> subscriptions), based on the <a>Protocol Bindings</a>.
+            Make a request to the underlying platform to initialize the
+            <a>Protocol Bindings</a> and then start serving external requests
+            for <a>WoT Interactions</a> (read, write and observe <a>Properties</a>,
+            invoke <a>Action</a>s and manage <a>Event</a> subscriptions),
+            based on the <a>Protocol Bindings</a>.
+            If for any <a>WoT Interaction</a> there is no |form| specified that
+            is supported by the implementation, reject |promise| with a
+            {{NotSupportedError}} and abort these steps.
+            Implementations MAY reject this step for any reason (e.g. if they
+            want to enforce further checks and constraints on interaction forms).
```

Comment:
These statements make it difficult to expose a TD that points to interactions *not* hosted by this runtime (maybe even with bindings that are not supported by this WoT runtime)

Reply:
Fixed.